specs/vm/instruction_set.md

```diff
@@ -702,6 +705,20 @@ Otherwise, advance the program counter `$pc` by `4`.
 
 See also: [BIP-65](https://github.com/bitcoin/bips/blob/master/bip-0065.mediawiki) and [Bitcoin's Time Locks](https://prestwi.ch/bitcoin-time-locks).
 
+### J: Jump
```

Comment:
This is the only instruction with a single letter. What if we name it `JMP` or `JRE` (jump register)?

Reply:
I want to avoid `JR*` since we may want jump-relative in the future. `JMP` might not be bad.

Reply:
Fixed in [e5b39af](https://github.com/FuelLabs/fuel-specs/pull/338/commits/e5b39af7bcd5ec57fe6d7b63c2c70a4d7e0f3b00)